packages/menu/src/styles.ts

```diff
@@ -305,7 +305,7 @@ export const disabledTextStyle: Record<Theme, string> = {
  */
 export const focusedMenuItemContainerStyle: Record<Theme, string> = {
   [Theme.Light]: css`
-    &:focus-visible {
+    &:focus {
```

Comment:
not as familiar with focus styling so looking to learn more, do we generally want to rely on the `:focus` selector over `:focus-visible`? are there general guidelines we follow for different cases?